Scripts/application.py

```diff
@@ -0,0 +1,58 @@
+import sqlite3
+import json
+from typing import Dict
+from flask import Flask
+app = Flask(__name__)
+
+@app.route('/')
+def hello_world():
+    return 'Hello World!'
+    
+@app.route('/api/v1/person')
+def return_people():
+    # returns a json of all people
+    with sqlite3.connect('../db/allYourSantaAreBelongToUs.db') as connection:
+        cursor = connection.cursor()
+        #get all of the people
+        cursor.execute('SELECT * FROM people')
+        raw_data: List = cursor.fetchall()
+        data: Dict = {}
+        for person in raw_data:
+            data[person[0]] = {"email": person[1], "assignment ID": person[2]}
+        json_data = json.dumps(data)
+    return json_data
+    
+@app.route('/api/v1/person/<person_id>')
+def return_person(person_id):
+    # returns a json of <person_id>
+    with sqlite3.connect('../db/allYourSantaAreBelongToUs.db') as connection:
+        cursor = connection.cursor()
+        cursor.execute(f'SELECT * FROM people WHERE person_ID={person_id}')
+        raw_data: List = cursor.fetchall()

```

Comment:
you need to consider that anyone can hit this api at any time. it's very possible someone will send a bogus id in which case raw_data is empty and you need to handle that correctly.

Reply:
I dealt with this so that the page will still return something. Let me know if this is not what you intended.